.github/workflows/release.yml

```diff
@@ -489,7 +489,7 @@ jobs:
           make release-cli
 
       - name: CLI - create release with binaries
-        uses: softprops/action-gh-release@v1
+        uses: softprops/action-gh-release@v2
         with:
           draft: true
           files: |
```

Comment:
The `env` key below is deprecated, see https://github.com/softprops/action-gh-release?tab=readme-ov-file#environment-variables
I don't think it's needed anymore as the `token` key defaults to `${{ github.token }}`, see https://github.com/softprops/action-gh-release?tab=readme-ov-file#inputs